src/main/kotlin/com/openlattice/linking/graph/PostgresLinkingQueryService.kt

```diff
@@ -293,35 +264,27 @@ class PostgresLinkingQueryService(private val hds: HikariDataSource, private val
 
     override fun getEntityKeyIdsOfLinkingIds(
             linkingIds: Set<UUID>,
-            normalEntitySetIds: Set<UUID>?
-    ): PostgresIterable<Pair<UUID, Set<UUID>>> {
-        return PostgresIterable(
-                Supplier {
-                    val connection = hds.connection
-                    val ps = connection.prepareStatement(buildEntityKeyIdsOfLinkingIdsSql(normalEntitySetIds != null))
-                    val linkingIdsArray = PostgresArrays.createUuidArray(connection, linkingIds)
-
-                    ps.setArray(1, linkingIdsArray)
-                    if (normalEntitySetIds != null) {
-
-                        /* Note: this inclusion may or may not speed up the function, depending how many partitions are
-                           covered by all the normal entity sets requested */
-                        val allPartitionsOfNormalEntitySets = partitionManager.getPartitionsByEntitySetId(normalEntitySetIds).values.flatten()
-
-                        ps.setArray(2, PostgresArrays.createUuidArray(connection, normalEntitySetIds))
-                        ps.setArray(3, PostgresArrays.createIntArray(connection, allPartitionsOfNormalEntitySets))
-                    }
-                    val rs = ps.executeQuery()
-                    StatementHolder(connection, ps, rs)
-                },
-                Function<ResultSet, Pair<UUID, Set<UUID>>> {
-                    val linkingId = ResultSetAdapters.linkingId(it)
-                    val entityKeyIds = ResultSetAdapters.entityKeyIds(it)
-                    linkingId to entityKeyIds
-                })
+            normalEntitySetIds: Set<UUID>
```

Comment:
Are there corresponding changes in linker to include `normalEntitySetIds` in this call?

Reply:
Yes, totally forgot to push up the PR. Thanks!

Reply:
https://github.com/openlattice/linker/pull/39

Reply:
:D